### PR TITLE
KAA-502: Add Cassandra to sandbox

### PR DESCRIPTION
**Pull request by batytskyy**

### CODE REVIEW

File: sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java [hunk 1]
```diff
@@ -344,6 +344,10 @@ protected void initBoxData() throws Exception {
             }
         }
 
+        LOG.info("Executing Cassandra cql script...");
+        executeSudoSsh("cqlsh -f "+CASSANDRA_INIT_SCRIPT);
```
Comment (ashvayka) [inline sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java:349]: spaces (formatting)

File: sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java [hunk 2]
```diff
@@ -344,6 +344,10 @@ protected void initBoxData() throws Exception {
...
+        Thread.sleep(5000L);
```
Comment (ashvayka) [inline sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java]: Why do we need this?

**Comment by ashvayka** [comment]
Some minor questions, otherwise, LGTM

File: sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java [hunk 3]
```diff
@@ -344,6 +344,10 @@ protected void initBoxData() throws Exception {
             }
         }
 
+        LOG.info("Executing Cassandra cql script...");
```
Comment (ashvayka) [inline sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java:348]: We need to control AbstractSandboxBuilder behavior using arguments. In other words, we need a dedicated argument that will trigger either Cassandra setup script or Mongo DB or both (all).

File: sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java (hunk 2 above)
Comment (batytskyy) [inline sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java]: It is approximate time of Cassandra init script execution.

**Pull request merged**